@xen-orchestra/lite/src/stories/form-input.story.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ComponentStory
+    :params="[
+      colorProp(),
+      iconProp('before').type('IconDefinition | string'),
+      iconProp('after').type('IconDefinition | string'),
+      model().type('string').required(),
+      prop('right').bool().widget(),
+      prop('disabled').bool().widget(),
+      prop('wrapper-attrs')
+        .obj('HTMLAttributes')
+        .widget()
+        .preset({ foo: 'bar' }),
+      prop('before-width').str().widget(),
+      prop('after-width').str().widget(),
+    ]"
+    :presets="presets"
+    v-slot="{ properties }"
+  >
+    <FormInput v-bind="properties" />
+  </ComponentStory>
+</template>
+
+<script lang="ts" setup>
+import { faDollarSign } from "@fortawesome/free-solid-svg-icons";
+import ComponentStory from "@/components/component-story/ComponentStory.vue";
+import FormInput from "@/components/form/FormInput.vue";
+import { colorProp, iconProp, model, prop } from "@/libs/story/story-param";
+
+const presets = {
+  $100: {
+    props: {
+      "model-value": "100",
```

Comment:
Right, I'll fix it.

I probably will need to make some checks about that because it clearly no longer works with `model-value` instead of `modelValue`, but when I wrote the code it was the opposite 🤔